app/src/main/java/com/sopt/geonppang/presentation/login/LoginActivity.kt

```diff
@@ -0,0 +1,42 @@
+package com.sopt.geonppang.presentation.login
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import com.sopt.geonppang.R
+import com.sopt.geonppang.databinding.ActivityLoginBinding
+import com.sopt.geonppang.presentation.SignActivity
+import com.sopt.geonppang.presentation.signup.SignUpActivity
+import com.sopt.geonppang.util.binding.BindingActivity
+import com.sopt.geonppang.util.extension.hideKeyboard
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class LoginActivity :
+    BindingActivity<ActivityLoginBinding>(R.layout.activity_login) {
+    private val viewModel: LogInViewModel by viewModels()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this
+
+        addListener()
+    }
+
+    private fun addListener() {
+        binding.root.setOnClickListener {
+            hideKeyboard(it)
+        }
+        binding.tvGotoSignup.setOnClickListener {
+            moveToSignup()
+        }
+        binding.toolbar.ivBack.setOnClickListener {
+            startActivity(Intent(this, SignActivity::class.java))
```

Comment:
```suggestion
            finish()
```
로 변경 이렇게 하면 스택에서 loginActivity가 제거 되지 않기때문에 이동한 SignActivity에서 뒤로가기를 하면 loginActivity로 이동됩니다.